docs/stcal/ramp_fitting/description.rst

```diff
@@ -19,6 +18,11 @@ saturation flags are found. Pixels are processed simultaneously in blocks
 using the array-based functionality of numpy.  The size of the block depends
 on the image size and the number of groups.
 
+There is a likelihood algorithm implemented based on Timothy Brandt's papers:
+`Brandt (2024) <https://iopscience.iop.org/article/10.1088/1538-3873/ad38d9>`__
+and
+`Brandt (2024) <https://iopscience.iop.org/article/10.1088/1538-3873/ad38da>`__.
+
```

Comment:
Sorry, I think maybe I wasn't clear in a previous comment.  I think these paper links should go in the detailed section below, where they were before.  In this overview section, we should just add a link to the later description like "see :ref:`likelihood_algorithm`".  

Also, please restore here the note that this algorithm is still being tested for scientific validity.

Reply:
Thanks for moving the paper links.  

Can you please add back the note about the algorithm still being tested?